src/libpmem/pmem.h

```diff
@@ -42,6 +42,45 @@ void pmem_init(void);
 
 int is_pmem_detect(const void *addr, size_t len);
 
+#ifndef AVX512F_AVAILABLE
```

Comment:
Why do we need to expose those functions? I don't see them being used outside pmem.c module.

Reply:
What do you mean? We are not exposing them.